### ISSUE TITLE
New screenshots view in Translations

### ISSUE THREAD

**Issue by marketachalupnikova** [tolgee/tolgee-platform#2138]
In most of the cases, there is no need for more than just 1 screenshot. At the moment, I have to click 2 times to close the screenshot windows.

**Comment by Totalus** [comment]
I agree, the screenshots view could be improved! I was about to write a new issue regarding this.

### User Story

As a translator, I want to be able to see the context of the text I am about to translate without loosing the focus on the translated text field.

### Suggested UI

I think it would fit pretty well on the right bar.

![image](https://github.com/user-attachments/assets/de3c4d35-eb5d-49f3-8175-ba036cea4958)

**Remarks**
- This would probably not require any major UI changes since the right bar already exists.
- Making the right bar resizable (like the left bar) would allow the user to optimize the screenshot size for better viewing.
- Clicking on the spotlighted image would zoom it.
- If more than one image is available, it could be selected by clicking on its thumbnail.
- If more than one image is available, there could optionally also be arrows that appear when hovering the spotlighted image that allow to switch to the next image.

### Additional comment

> In most of the cases, there is no need for more than just 1 screenshot.

There are still come use cases where more than one screenshots could be useful, for strings that appear in more than one location in the app.

**Comment by ZuzanaOdstrcilova** [comment]
Design is ready: https://www.figma.com/design/wVmz098ZrSxswOI0nbIjQ2/Design-system-Tolgee?node-id=3268-15315&t=WGI8P2fmJbOzA6WE-1

Waiting for implementation.
